spec/helpers/cats_helper_spec.rb

```diff
@@ -11,5 +11,5 @@
 #   end
 # end
 RSpec.describe CatsHelper, type: :helper do
-  pending "add some examples to (or delete) #{__FILE__}"
+
```

Comment:
This file can be deleted. We won't need it and it just gets in the way when running tests.